core/src/createTransaction.ts

```diff
@@ -15,11 +15,28 @@ export class CreateTransactionError extends Error {
 }
 
 export interface CreateTransactionParams {
+    /** The mint address of the SPL token */
     splToken?: PublicKey;
+    /** An array of public keys used to identify this transaction */
     reference?: PublicKey | PublicKey[];
+    /** Creates an additional transaction for the [Memo Program](https://spl.solana.com/memo) */
     memo?: string;
 }
 
+/**
+ * Create a transaction
+ *
+ * **Reference** implementation for wallet providers.
+ *
+ * @param connection - A connection to the cluster.
+ * @param payer - `PublicKey` of the payer.
+ * @param recipient - The **native SOL** address the payment should be made to.
+ * @param amount - The amount of SOL or SPL token that should be transferred.
+ * @param {CreateTransactionParams} createTransactionParams - Additional parameters
+ * @param createTransactionParams.splToken - The mint address of the SPL token.
+ * @param createTransactionParams.reference - An array of public keys used to identify this transaction.
+ * @param createTransactionParams.memo - Creates an additional transaction for the [Memo Program](https://spl.solana.com/memo)
```

Comment:
It's not an additional transaction, but an additional instruction on the same transaction